lib/helpers/HealthCheckUiHelper.ts

```diff
@@ -0,0 +1,65 @@
+import {Page, Locator, expect} from "@playwright/test";
+import {UiBaseLocators} from "./UiBaseLocators";
+
+export class HealthCheckUiHelper extends UiBaseLocators {
+  private readonly healthCheckTab: Locator;
+  private readonly healthCheckGroupBox: Locator;
+  private readonly performanceAllChecksBtn: Locator;
+  private readonly positiveTag: string;
+  private readonly warningTag: string;
+  private readonly dangerTag: string;
+  private readonly headline: Locator;
+  private readonly healthCheckGroup: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.healthCheckTab = page.getByRole('tab', {name: 'Health Check'});
+    this.healthCheckGroupBox = page.locator('umb-health-check-group-box-overview');
+    this.performanceAllChecksBtn = page.getByLabel('Perform all checks');
+    this.positiveTag = 'uui-tag[color="positive"]';
+    this.warningTag = 'uui-tag[color="warning"]';
+    this.dangerTag = 'uui-tag[color="danger"]';
```

Comment:
Unfortunately, the code will likely return an error if you write like this: "this.healthCheckGroupBox.filter({has: this.page.getByText(healthCheckGroupName)}).positiveTag" since we cannot combine 2 Locators.